python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -40,6 +40,14 @@
 from tests.util.date import DateRange
 
 
+if sys.platform == "win32":
```

Comment:
Can we not test if the `CI_MONGO_HOST` env var is set? If we don't, what happens if you run this locally on Linux?

Reply:
No we shouldn't, since we don't insist on that being set for local runs (which work fine using pytest-server-fixtures as long as `mongod` is installed).